articles/idam/idam.md

```diff
@@ -0,0 +1,83 @@
+---
+title: ID & Access Management API | UKCloud Ltd
+description: Provides information on managing permissions using the ID & Access Management API
+services: idam
```

Comment:
For articles that apply to multiple services, we use other rather than create new service ids.

Reply:
Resolved in b07515a.